task_sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -43,6 +44,7 @@
     SetRenderedFields,
     SetXCom,
     StartupDetails,
+    SucceedTask,
```

Comment:
Lets be a bit more consistent with `TaskState` here:
```suggestion
    TaskSuccess,
```

Reply:
I was aligning with `DeferTask`, `RescheduleTask` etc. I am OK making this change but should we do for remaining states too?

Reply:
Oh yeah, `SucceedTask` is better then.

Reply:
Leave them as you have them